packages/console-snake/src/commands/play-command.js

```diff
@@ -25,7 +25,13 @@ export default async function main({
   readline.createInterface(process.stdin, process.stdout)
 
   for (let i = 1; ; ++i) {
-    const {board: newBoard, hasCollided} = snake.executeTick(board, i % numTicksToLengthen === 0)
+    const isLengthen = i % numTicksToLengthen === 0
+    const {board: newBoard, hasCollided, isAlive} = snake.executeTick(board, isLengthen)
+    if (!isAlive) {
+      console.clear()
+      console.log('God Bless The Snake Soul')
+      process.exit()
```

Comment:
`process.exit` is very "hard" and means that this module would not be amenable to testing (yes, it's not tested, but could be in the future).

Thus, a simple `return` would be nicer here, and would have the same effect (I think...)